src/bootstrap/bootstrap.py

```diff
@@ -285,45 +314,78 @@ def fix_executable(self, fname):
         try:
             subprocess.check_output(
                 ["patchelf", "--set-interpreter", correct_interpreter, fname])
-        except subprocess.CalledProcessError as e:
-            print("warning: failed to call patchelf: %s" % e)
+        except subprocess.CalledProcessError as reason:
+            print("warning: failed to call patchelf:", reason)
```

Comment:
You removed the `%s` on this print, but not the others - I don't think it matters, but it's probably worth being consistent.